cs/client.py

```diff
@@ -73,6 +74,20 @@
 FAILURE = 2
 
 
+def check_key(key, allowed):
+    """
+    Validate that the specified key is allowed according the provided
+    list of patterns.
+    """
+
+    from fnmatch import fnmatch
```

Comment:
The import should be at the top, with the others

Reply:
Fixed.